sinks/webhook/webhook.go

```diff
@@ -95,8 +96,10 @@ func (ws *WebHookSink) Send(event *v1.Event) (err error) {
 	defer resp.Body.Close()
 
 	if resp != nil && resp.StatusCode != http.StatusOK {
-		klog.Errorf("failed to send msg to sink, because the response code is %d", resp.StatusCode)
-		return fmt.Errorf("failed to send msg to sink, because the response code is %d", resp.StatusCode)
+		body, _ := ioutil.ReadAll(resp.Body)
```

Comment:
Add error handler to ioutil.ReadAll please.

Reply:
updated